index.html

```diff
@@ -320,8 +320,8 @@
                             this.stream = queryStream;
                         } else {
                             searchParams.set('stream', this.stream);
+                            history.pushState(null, null, `${window.location.origin}?${searchParams.toString()}`);
```

Comment:
Did you see https://github.com/coreos/fedora-coreos-browser/pull/11#discussion_r410427038 ?

Reply:
Yes, exactly that. Should considering the route `/browser` in this line. Worked locally since there's no routing.

Reply:
Thanks for pointing that out, will update